packages/itwinui-css/src/avatar/avatar.scss

```diff
@@ -48,7 +48,7 @@
     font-size: var(--_iui-avatar-font-size);
   }
 
-  > .iui-icon {
+  svg {
```

Comment:
Is this considered a breaking change for our CSS pkg?

Reply:
Ohh, right, it looks like a breaking change. It also breaks the avatar demo html page.

![image](https://user-images.githubusercontent.com/45748283/215902478-27c6a50c-0883-4938-a3cc-b91121547e86.png)

I auto-committed the suggestion (https://github.com/iTwin/iTwinUI/pull/1040#discussion_r1092530211), but I think it needs some editing. Will look into it more.

Reply:
Would this be alright? This keeps the html demos the same, is not a breaking change, and also targets svgs with `.iui-icon` which is likely how icons are passed on the react side.

```suggestion
  > .iui-icon, > svg {
```

Reply:
looks like you need to add these three lines to fix the positioning

```
left: 50%;
top: 50%;
transform: translate(-50%, -50%);
```

and then the wrapping section would need `display: flex; align-items: end`

alternatively, `display: grid` could be used instead of `position: absolute` to center the svg. I think this would avoid the need to set styles on the wrapper.

i do not think it would be a breaking change because anyone using `iui-icon` will be unaffected.

Reply:
Thanks, the first option you mentioned worked. Now targeting the avatar's `svg` instead of `> .iui-icon` ([`7a2f509` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/7a2f509a9d896b0b183b8479771fc19e010b0ba2))